templates/infinispan-ephemeral.json

```diff
@@ -10,28 +10,37 @@
       "iconClass": "icon-infinispan",
       "tags": "java,database,datagrid,jboss",
       "template.openshift.io/documentation-url": "http://infinispan.org/documentation/",
-      "template.openshift.io/long-description": "Infinispan is a high performance, scalable, key/value data grid solution.",
+      "template.openshift.io/long-description": "This template provides a standalone Infinispan server (a high performance, scalable, key/value data grid solution) without persistence (in other words your data will be lost upon restart). In order to provide custom configuration, please update $ApplicationName-configuration ConfigMap and restart a Pod.",
       "template.openshift.io/provider-display-name": "Red Hat, Inc.",
       "template.openshift.io/support-url": "http://infinispan.org"
     },
+    "labels": {
+      "template": "infinispan-ephemeral"
+    },
     "name": "infinispan-ephemeral"
   },
   "objects": [
     {
       "kind": "ServiceAccount",
       "apiVersion": "v1",
       "metadata": {
+       "labels": {
+          "template": "infinispan-ephemeral"
+        },
```

Comment:
you don't need to do this.  all objects in the template will get whatever labels you've defined at the template level.